Edge/edgegateway_mqtt.c

```diff
@@ -36,12 +36,13 @@ static struct thing *registered_things[10];
 static int registered_thing_counter = 0;
 static char http_ack_context[65];
 static char client_id[50];
+static char password[65];
```

Comment:
Let's make sure this much space is ok

Reply:
Yeah this much space is ok for hmac, we used same space for events as well.